package.json

```diff
@@ -4,51 +4,51 @@
   "version": "1.0.1",
   "main": "lib/index.js",
   "dependencies": {
-    "@oclif/core": "^1.16.0",
-    "@salesforce/command": "^5.2.6",
-    "@salesforce/core": "^3.26.2",
-    "@salesforce/kit": "^1.6.0",
+    "@oclif/core": "^1.22.0",
+    "@salesforce/command": "^5.2.33",
+    "@salesforce/core": "^3.32.11",
+    "@salesforce/kit": "^1.8.0",
     "semver-compare": "^1.0.0",
     "signal-exit": "^3.0.6",
-    "tslib": "^2.3.1"
+    "tslib": "^2.4.1"
   },
   "devDependencies": {
-    "@oclif/plugin-command-snapshot": "^3.1.3",
-    "@oclif/plugin-commands": "^2.1.0",
-    "@oclif/plugin-help": "^5.1.12",
-    "@salesforce/cli-plugins-testkit": "^2.3.13",
+    "@oclif/plugin-command-snapshot": "^3.2.13",
+    "@oclif/plugin-commands": "^2.2.1",
+    "@oclif/plugin-help": "^5.1.20",
+    "@salesforce/cli-plugins-testkit": "^3.2.14",
```

Comment:
Not sure what changed here. 🤷 https://github.com/salesforcecli/cli-plugins-testkit/commit/ccdf5c08547922552b2b28d33a46bb49392f6d5e

Reply:
OK, looks like this breaks our tests, because we rely on `setupCommands` which is replaced by `createOrgs`. But I don't see how we can use `createOrgs` instead; our `setupCommands` is doing something different:

https://github.com/salesforce/sfdx-plugin-lwc-test/blob/11a4b63bf6264897774bf5a78b224c14a0a1ca37/test/commands/lwc/test/run.nut.ts#L24